project/Dependencies.scala

```diff
@@ -6,7 +6,7 @@ object Dependencies {
 
   val scala210 = "2.10.7"
   val scala211 = "2.11.12"
-  val scala212 = "2.12.10"
+  val scala212 = "2.12.11"
   val scala213 = "2.13.1"
```

Comment:
Thanks for the various fixes. Scala 2.13.2 has been released this week, do we want to update that too?

Reply:
Yeah, good thinking.